<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Meta</class>
 <widget class="QWidget" name="Meta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>196</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Meta Info</string>
  </property>
  <widget class="QCheckBox" name="b_modify">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modified save</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="b_modify_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save using modified content</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="n_creads">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="n_cwrites">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="n_cloads">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="n_csaves">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reads</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Writes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Context</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_savename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="i_lmessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_mode">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>10</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="prefix">
    <string>Mode </string>
   </property>
   <property name="maximum">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="n_version">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>4</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Hex</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
